lib/omnibus/cli.rb

```diff
@@ -61,12 +61,32 @@ def execute!
     #
     #   $ omnibus build chefdk
     #
+    method_option :output_manifest,
+      desc: "Create version-manifest.json in current directory at the end of the build",
+      type: :boolean,
+      default: false
+    method_option :use_manifest,
+      desc: "Use the given manifest when downloading software sources.",
+      type: :string,
+      default: nil
     desc 'build PROJECT', 'Build the given Omnibus project'
     def build(name)
-      project = Project.load(name)
+      manifest = if @options[:use_manifest]
+                   Omnibus::Manifest.from_file(@options[:use_manifest])
+                 else
+                   nil
+                 end
 
+      project = Project.load(name, manifest)
       say("Building #{project.name} #{project.build_version}...")
-      project.build_me
+      project.download
+      project.build
+
+      if @options[:output_manifest]
+        File.open('version-manifest.json', 'w') do |f|
+          f.write(JSON.pretty_generate(project.built_manifest.to_hash))
+        end
+      end
```

Comment:
@stevendanna Should we be generating a 2nd manifest here? It would be safer to either drop this option and copy the one generated by `Omniubs::Project#build` into place. This would work similar to what we do with packages:
https://github.com/chef/omnibus/blob/master/lib/omnibus/project.rb#L1046-L1049

I have some mixed feelings about this approach but at least things would be consistent.